packages/notifi-react-card/lib/context/NotifiSubscriptionContext.tsx

```diff
@@ -78,6 +90,10 @@ export const NotifiSubscriptionContextProvider: React.FC<
   const [email, setEmail] = useState<string>('');
   const [phoneNumber, setPhoneNumber] = useState<string>('');
   const [telegramId, setTelegramId] = useState<string>('');
+  const [demoPreview, setDemoPreview] = useState<DemoPreview | null>({
+    view: 'expired',
+    data: defaultDemoConfigV1,
+  });
```

Comment:
i'd feel better if the default value is null

Reply:
Already Migrated demoPreview to newly created 'NotifiDemoPreviewContext' that handles demoPreview related context which contain new default value as null.